webpack/assets/javascripts/react_app/components/HostsIndex/Columns/core.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { translate as __ } from '../../../common/I18n';
+import RelativeDateTime from '../../common/dates/RelativeDateTime';
+import forceSingleton from '../../../common/forceSingleton';
+
+const coreHostsIndexColumns = [
+  {
+    columnName: 'name',
+    title: __('Name'),
+    wrapper: ({ name }) => <Link to={`hosts/${name}`}>{name}</Link>,
+    isSorted: true,
+    weight: 0,
+  },
+  {
+    columnName: 'hostgroup',
+    title: __('Host group'),
+    wrapper: hostDetails => hostDetails?.hostgroup_name,
```

Comment:
AFAIK today the hostgroup is a link.

Reply:
I can make it a link, no problem. For now I was just going off the mockup.